wicketd/src/http_entrypoints.rs

```diff
@@ -48,6 +53,12 @@ pub fn api() -> WicketdApiDescription {
     api
 }
 
+#[derive(Clone, Copy, Debug, Serialize, Deserialize, JsonSchema, PartialEq)]
+pub struct ForceRefresh {
```

Comment:
I think this should be called something like `GetInventoryParams`.

Reply:
Good call; renamed in b3976eac